Client/RecommendationRequestClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Client;
+
+interface RecommendationRequestClient
+{
+    /**
+     * @param int $userId
+     * @param int $scenarioId
+     * @param int $locationId
+     * @param int $limit
+     * @return string|null JSON response
```

Comment:
If you didn't add a interface it would maybe have been fine, but when adding interface it would be more appropriate to abstract the YC response somehow.

Also, further below it seems that it is not a json string returned here, but rather a array hash.